tests/boulder-start.sh

```diff
@@ -5,5 +5,6 @@ export GOPATH="${GOPATH:-/tmp/go}"
 
 go get -d github.com/letsencrypt/boulder
 cd $GOPATH/src/github.com/letsencrypt/boulder
+./test/create_db.sh
```

Comment:
It's in Boulder's repo: https://github.com/letsencrypt/boulder/blob/master/test/create_db.sh